feature/bgp/tests/local_bgp_test/local_bgp_test.go

```diff
@@ -132,27 +169,33 @@ func TestDisconnect(t *testing.T) {
 	ateIP := ateAttrs.IPv4
 	dutIP := dutAttrs.IPv4
 	nbrPath := statePath.Neighbor(ateIP)
+	// Configure Network instance type, it has to be configured explicitly by user.
+	configureNIType(t)
 
 	// Clear any existing config
-	dutConfPath.Delete(t)
-	ateConfPath.Delete(t)
+	gnmi.Delete(t, dut, dutConfPath.Config())
+	gnmi.Delete(t, ate, ateConfPath.Config())
 
 	// Apply simple config
 	dutConf := bgpWithNbr(dutAS, "", &telemetry.NetworkInstance_Protocol_Bgp_Neighbor{
 		PeerAs:          ygot.Uint32(dutAS),
 		NeighborAddress: ygot.String(ateIP),
+		PeerGroup:       ygot.String(peerGrpName),
 	})
 	ateConf := bgpWithNbr(dutAS, "", &telemetry.NetworkInstance_Protocol_Bgp_Neighbor{
 		PeerAs:          ygot.Uint32(dutAS),
 		NeighborAddress: ygot.String(dutIP),
+		PeerGroup:       ygot.String(peerGrpName),
 	})
-	dutConfPath.Replace(t, dutConf)
-	ateConfPath.Replace(t, ateConf)
-	nbrPath.SessionState().Await(t, time.Second*15, telemetry.Bgp_Neighbor_SessionState_ESTABLISHED)
+	gnmi.Replace(t, dut, dutConfPath.Config(), dutConf)
+	gnmi.Replace(t, ate, ateConfPath.Config(), ateConf)
+	time.Sleep(time.Second * 15)
```

Comment:
Why sleep for 5 seconds before awaiting? Can you just add 5 seconds to the await call?